### PR TITLE
Remove obsolete directories in directory layout

### PR DESCRIPTION
**Pull request by skim1420**
@ewiseblatt

### CODE REVIEW

File: google/README.txt [hunk 1]
```diff
@@ -11,21 +11,18 @@ our development, so are not necessarily generalized or applicable outside GCE.
 General layout is this:
 
 google/
-   cassandra   Cassandra schemas, perhaps these could go to config.
-
-   config      Our *-local.yml overrides and default master configuration.
-               These are installed as "config_templates".
+   codelab     (Ignore) Supporting script for building codelab image
```
Comment (ewiseblatt) [inline google/README.txt]: is there a url for the codelab? This would be a good place to reference it.

File: google/README.txt [hunk 2]
```diff
@@ -11,21 +11,18 @@ our development, so are not necessarily generalized or applicable outside GCE.
...
 
    dev         Scripts meant for development support, not operational usage.
                These are not included in a typical deployment, though are
                still open source for developers.
 
-   install     These are release installation scripts. They are kept separate
-               from the runtime admin scripts for convenience.
+   google_cloud_logging
+               (Optional) Script to install Google Cloud Logging and
+               Monitoring integration.
```
Comment (ewiseblatt) [inline google/README.txt]: logging only, not monitoring.

File: google/README.txt [hunk 3]
```diff
@@ -11,21 +11,18 @@ our development, so are not necessarily generalized or applicable outside GCE.
...
 
-   pylib       A python package containing modules used by the runtime scripts
-               All these are part of the runtime environment. The developer
-               scripts and tools put their python in the dev directory.
+   stackdriver_monitoring
+               (Optional) Configurations to enable StackDriver integration.
```
Comment (ewiseblatt) [inline google/README.txt]: That was the original intent, but monitoring is baked into spinnaker itself. Instead I have supporting tools in here. The tools are development oriented for the time being. Maybe this is "administrative support for Spinnaker Stackdriver integration".

**Comment by ewiseblatt** [comment]
LGTM

File: google/README.txt (hunk 1 above)
Comment (skim1420) [inline google/README.txt]: Hmm.. not sure if that's the objective here, and for that reason I explicitly call out (Ignore) on this dir.

**Pull request merged**